test/matchers/utils.test.ts

```
@@ -0,0 +1,420 @@
+    expect(result).toContain('"specs":{"weight":"2kg"');
+    expect(result).toContain('"dimensions":{"width":10,"height":20}');
+    expect(result).toContain('[{"a":1},{"b":2}]');
+    expect(result).toContain('"tags":["electronics","gadget"]');
+    expect(result).toContain('string item');
+  });
+});
+
+describe('PROMPTFOO_DISABLE_OBJECT_STRINGIFY environment variable', () => {
+  afterEach(() => {
+    // Clean up environment variable after each test
+    delete process.env.PROMPTFOO_DISABLE_OBJECT_STRINGIFY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)